sdks/wasm/src/library_mono.js

```diff
@@ -366,6 +414,45 @@ var MonoSupportLib = {
 			return { __value_as_json_string__: JSON.stringify (res_details) };
 		},
 
+		_get_ptr_args: function (objectId) {
+			var parts = this._split_object_id (objectId);
+			if (parts.length != 3)
+				throw new Error (`Bug: Unexpected objectId format for a pointer, expected 3 parts: ${objectId}`);
+			return JSON.parse (parts [2]);
+		},
+
+		_get_updated_ptr_id: function (objectId, new_args) {
+			var old_args = {};
+			if (objectId != undefined && objectId != '')
```

Comment:
i'd prefer something like ```(typeof(objectId) === 'string') && objectId.length``` but the current code is fine, just kind of awkward. (is any type fine as long as it's not undefined and it's not ''? or is it supposed to be a string)